### PR TITLE
Merge forked changes back upstream

### PR DESCRIPTION
**Pull request by elliottsj**
These are all of the changes we've made to our fork of next-routes, requesting a merge back upstream.

Most of these are dependency upgrades (using https://renovatebot.com/ instead of Greenkeeper), but there are also some updates to config, tests, and documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Dynamic Routes for Next.js
 
-[![npm version](https://d25lcipzij17d.cloudfront.net/badge.svg?id=js&type=6&v=1.4.2&x2=0)](https://www.npmjs.com/package/next-routes) [![Coverage Status](https://coveralls.io/repos/github/fridays/next-routes/badge.svg)](https://coveralls.io/github/fridays/next-routes) [![Build Status](https://travis-ci.org/fridays/next-routes.svg?branch=master)](https://travis-ci.org/fridays/next-routes)
+**NOTICE: This is a fork of [fridays/next-routes](https://github.com/fridays/next-routes). This will continue to be maintained until [fridays/next-routes](https://github.com/fridays/next-routes) becomes maintained again. See https://github.com/fridays/next-routes/issues/244.**
```
Comment (emileber) [inline README.md]: This should be removed before being merged here.

File: README.md [hunk 2]
```diff
@@ -13,29 +15,37 @@ Easy to use universal dynamic routes for [Next.js](https://github.com/zeit/next.
 Install:
 
 ```bash
-npm install next-routes --save
+npm install @yolkai/next-routes --save
```
Comment (emileber) [inline README.md]: Also this wouldn't be the right package if merged here.

File: test/index.test.js [hunk 3]
```diff
@@ -2,41 +2,41 @@
 import React from 'react'
 import ReactShallowRenderer from 'react-test-renderer/shallow'
 import NextLink from 'next/link'
-import nextRoutes from '../dist'
+import nextRoutes from '..'
```
Comment (emileber) [inline test/index.test.js]: Shouldn't the tests be done with the built files?

**Comment by emileber** [comment]
Not sure what's the merge strategy by default on this project, but hopefully it won't merge all 128 commits and instead will squash and merge?

**Comment by fridays** [comment]
Thanks a lot @elliottsj that looks good like a great start. I'm currently working on other things and can't test it all out. We'll need some community feedback on the changes, @emileber spotted some good points. Please merge when you're ready, I'm going to add to to npm then so you can publish future updates.

**Pull request closed without merging**